src/main.ts

```diff
@@ -192,28 +193,32 @@ export async function activate(context: ExtensionContext) {
 
 	await reloadClient();
 
-	// Best way to determine package updates. Will work for npm, yarn, pnpm and bun. (Might work for more files also).
-	// It is not possible to listen node_modules, because it is usually gitignored.
-	const watcher = workspace.createFileSystemWatcher("**/*lock*");
-	context.subscriptions.push(
-		watcher.onDidChange(async () => {
-			try {
-				// When the lockfile changes, reload the biome executable.
-				outputChannel.appendLine("Reloading biome executable.");
-				if (client.isRunning()) {
-					await client.stop();
-				}
-				await reloadClient();
-				if (client.isRunning()) {
-					await client.restart();
-				} else {
-					await client.start();
+	if (workspace.workspaceFolders?.[0]) {
+		// Best way to determine package updates. Will work for npm, yarn, pnpm and bun. (Might work for more files also).
+		// It is not possible to listen node_modules, because it is usually gitignored.
+		const watcher = workspace.createFileSystemWatcher(
+			new RelativePattern(workspace.workspaceFolders[0], "*lock*"),
```

Comment:
We need to use a RelativePattern here, otherwise the watcher won't work as expected, which we want it to watch only the root.